src/components/Image/Image.st.css

```diff
@@ -60,32 +60,22 @@
   min-height: 100%;
 }
 
-.root.contain .absoluteImage,
-.root.contain .relativeImage {
+.root.contain .image {
```

Comment:
Since there weren't any differences between the two classes, I thought it would be easier to use just one class

Reply:
Before having RelativeMediaImage it was necessary but now I agree this is much better